cumulusci.yml

```diff
@@ -938,6 +938,71 @@ flows:
             2:
                 task: update_admin_profile
 
+    enable_gift_entry_managed:
+        description: Enable the Gift Entry feature in a scratch org (managed package only)
+        group: NPSP
+        steps:
+            1:
+                flow: enable_advanced_mapping_managed
+            2:
+                task: execute_anon
+                options:
+                    managed: true
+                    unmanaged: false
+                    apex: >
```

Comment:
We might as well create a task for this Apex since we're calling it twice.

Reply:
@lparrott I thought about this a little more, and I think it doesn't make as much sense to split this out into a task since gift entry shouldn't ever be enabled without first enabling advanced mapping, and splitting this out into its own task would imply it can be executed in isolation. Thoughts?

Reply:
Good point, there might be a way to write the managed version as an "overload" of the unmanaged, like you did with `enable_advanced_mapping_managed` and avoid using the apex block twice, but I wouldn't spend much time on that if it isn't straightforward!

Reply:
How about moving this apex into a file and using the `path:` param on these two tasks? The scripts directory has a few of these already.